10.9-libcxx/stable/main/finkinfo/graphics/libavcodec60-6.0-shlibs.info

```diff
@@ -110,9 +111,12 @@ Source-Checksum: SHA256(62fe9fd098cb537d4b61731b11049e1ac179f38c870a41b8e9d556af
 ###
 PatchFile: %n_texinfo72.patch
 PatchFile-MD5: 460868fa2b167e7c57f605d1018c4374
+PatchFile2: %n.patch
```

Comment:
would be less confusing to have that defined as `PatchFile` and the extra texinfo one as `PatchFile2`, but that's purely cosmetic.

Reply:
Not needed. Keep the patches in the order they are, please. if anything, rename PatchFile2 (%n.patch) to %n-incompatible-type.patch